modules/repository-s3/src/test/java/org/elasticsearch/repositories/s3/S3ClientSettingsTests.java

```diff
@@ -37,6 +37,7 @@ public void testThereIsADefaultClientByDefault() {
         assertThat(defaultSettings.protocol, is(Protocol.HTTPS));
         assertThat(defaultSettings.proxyHost, is(emptyString()));
         assertThat(defaultSettings.proxyPort, is(80));
+        assertThat(defaultSettings.proxyScheme, is(Protocol.HTTPS));
```

Comment:
`HTTP`

Reply:
Sure @kingherc, I will update that. Please confirm this goes same for `plugins/discovery-ec2/src/main/java/org/elasticsearch/discovery/ec2/Ec2ClientSettings.java`. Thanks!!

Reply:
Yes @heissanjay , both use ultimately https://github.com/aws/aws-sdk-java/blob/004bb46aa8b3912f172a1240c3f622336e167921/aws-java-sdk-core/src/main/java/com/amazonaws/ClientConfiguration.java#L171 which mentions a default value of HTTP for the proxy